### PR TITLE
use unified monitoring package

### PR DESCRIPTION
**Pull request by zirain**
**Please provide a description of this PR:**

### CODE REVIEW

File: pilot/pkg/bootstrap/monitoring.go [hunk 1]
```diff
@@ -45,12 +45,25 @@ var (
 		"istiod_uptime_seconds",
 		"Current istiod server uptime in seconds",
 	)
+
+	versionTag = monitoring.MustCreateLabel("version")
+	// Export pilot version as metric for fleet analytics.
```
Comment (hzxuzhonghu) [inline pilot/pkg/bootstrap/monitoring.go:50]: can you remove this comment

Comment (zirain) [reply pilot/pkg/bootstrap/monitoring.go]: done

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by zirain** [comment]
`pilot_info` was added by googler, can you ask anthos team?

**Comment by howardjohn** [comment]
It was 5 years ago, before Istiod merge is probably why. We have a general
problem of inconsistent metrics, etc. Hard to change though as are sort of
an API

**Pull request merged**